.changeset/afraid-insects-move.md

```diff
@@ -0,0 +1,7 @@
+---
+"@lens-protocol/api-bindings": minor
+"@lens-protocol/domain": minor
+"@lens-protocol/react": minor
```

Comment:
nit: would be better to just mention the package that this change affects the most. Right now changeset will generate the changelog mentioning `Added useProfileOwnedBy hook` in `api-bindings, domain and react` but in reality `useProfileOwnedBy` is only a part of `react` package, and the changes in `api-bindings,domain` are internal one 
(same applies to the `happy-socks-design` changeset)

Reply:
got it